plugins/modules/gitlab_project.py

```diff
@@ -172,6 +172,39 @@
       - This option is only used on creation, not for updates. This is also only used if I(initialize_with_readme=true).
     type: str
     version_added: "4.2.0"
+  builds:
+    description:
+      - C(private) Repository CI/CD allowed only to project members.
+      - C(disabled) Repository CI/CD is disabled.
+      - C(enabled) Repository CI/CD is enabled.
+    default: enabled
+    type: str
+    choices: ["private", "disabled", "enabled"]
+    aliases:
+      - builds_access_level
```

Comment:
Aliases should be avoided for new options, unless they are absolutely necessary. What is the reason for having an alias here?

Reply:
The aliases were introduced to clarify the gitlab api rest attributed managed, but you're right. I'll remove the aliases.

Reply:
Personally I think it'd be better if the attribute keys follow the GitLab API ones anyway, just as all of the other ones do.

Users will have to consult the upstream GitLab API docs in any case for some of these, even just to check the permissions or the type of token required. So it makes it easier to map to this module IMO.

Reply:
OK